### PR TITLE
i18n(es): update `rss`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Updates `rss` based on this commit 0d6725c26743194972cd04d46759e9fab0997f41

#### Related issues & labels (optional)

- Closes # N/A
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/guides/rss.mdx [hunk 1]
```diff
@@ -206,6 +206,31 @@ export function GET(context) {
 }
 ```
 
+## Removing trailing slashes
+
```
Comment (yanthomasdev) [inline src/content/docs/es/guides/rss.mdx:210]: Looks like you forgot to translate this heading!

Comment (dreyfus92) [reply src/content/docs/es/guides/rss.mdx:210]: Nice catch I will fix this shortly

File: src/content/docs/es/guides/rss.mdx [hunk 2]
```diff
@@ -206,6 +206,31 @@ export function GET(context) {
 }
 ```
 
+## Eliminando las barras diagonales finales
+
+El RSS feed de Astro produce enlaces con una barra diagonal al final por defecto, sin importar el valor que hayas configurado para `trailingSlash`. Esto significa que los enlaces de tu RSS pueden no coincidir exactamente con las URLs de tus posts.
```
Comment (Waxer59) [inline src/content/docs/es/guides/rss.mdx:211]:
```suggestion
El feed RSS de Astro produce enlaces con una barra diagonal al final por defecto, sin importar el valor que hayas configurado para `trailingSlash`. Esto significa que los enlaces de tu RSS pueden no coincidir exactamente con las URLs de tus posts.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia!

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Comment by yanthomasdev** [comment]
After we get the Steps component in here, LGTM https://github.com/withastro/docs/commits/main/src/content/docs/en/guides/rss.mdx?since=2024-05-02T17:28:28.000Z

**Comment by dreyfus92** [comment]
we cooking @yanthomasdev 👨🏻‍🍳

**Pull request merged**